ci-operator/step-registry/rosa/cluster/wait-ready/nodes/rosa-cluster-wait-ready-nodes-commands.sh

```diff
@@ -150,6 +150,19 @@ function getDesiredComputeCount {
     echo "--auto-scaling disabled, --replicas set, retrieving replica count desired"
     desired_compute_count=$(rosa describe cluster -c "$CLUSTER_ID"  -o json  |jq -r '.nodes.compute')
   fi
+  if [[ ! -z ${MP_NAME} ]];then
+    compute_count=$(rosa describe machinepool ${MP_NAME}  -c "$CLUSTER_ID"  -o json  |jq -r '.replicas')
+      if [[ "$compute_count" = "null" ]]; then 
+        echo "--auto-scaling enabled, retrieving min_replica count desired"
+        if [[ $HOSTED_CP = "true" ]];then
+          compute_count=$(rosa describe machinepool  ${MP_NAME} -c "$CLUSTER_ID" -o json  | jq -r '.autoscaling.min_replica') 
+        else
+          compute_count=$(rosa describe cluster -c "$CLUSTER_ID" -o json  | jq -r '.autoscaling.min_replicas*') 
```

Comment:
Is it right to describe the cluster? And there could be a compile error to use '*' in jq.